src/consolelogger.c

```diff
@@ -157,12 +157,12 @@ void consolelogger_log_with_GetLastError(const char* file, const char* func, int
     timeString = get_ctime(&t);
 #endif // LOGGER_DISABLE_PAL
 
-    systemMessage = printf_alloc("Error: Time:%.24s File:%s Func:%s Line:%d %s", timeString, file, func, line, lastErrorAsString);
+    systemMessage = printf_alloc("Error: Time:%.24s File:%s Func:%s Line:%d %s", timeString == NULL ? "<NO TIME IMPL>" : timeString, file, func, line, lastErrorAsString);
```

Comment:
just do this one on line 159 and not at every print?

timeString = (timeString == NULL) ? "<NO TIME IMPL>" : timeString;